java/demo/war/WEB-INF/appengine-web.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <application>libphonenumber</application>
+  <application>libphonenumber-hrd</application>
```

Comment:
I fixed that already, didn't I?
https://github.com/googlei18n/libphonenumber/blob/master/java/demo/war/WEB-INF/appengine-web.xml